index.html

```diff
@@ -9144,7 +9146,8 @@ <h3 id="example-2">
               A read-only decibel value for metering purposes, representing the
               current amount of gain reduction that the compressor is applying
               to the signal. If fed no signal the value will be 0 (no gain
-              reduction).
+              reduction). When this attribute is read, return the value of the
+              private slot [[\reduction]].
```

Comment:
IIRC, what you've done is to say [[reduction]] is ignored when the signal is present, and zero is returned instead.